aip/general/0121.md

```diff
@@ -53,6 +53,19 @@ database schema. In fact, having an API that is identical to the underlying
 database schema is actually an anti-pattern, as it tightly couples the surface
 to the underlying system.
 
+### Resource requirements
+
+For a valid resource-oriented API, the following must be satisfied:
+
+- The resource must have a unique [resource name][] by which it can be
+  identified and referenced.
+- The resource exposes only standard methods ([Get][], [List][], [Create][],
+  [Update][], [Delete][]) or [custom methods][].
+- The resource API must support [Get][] and [List][] operations.
```

Comment:
Copied from the other PR:
This language conflicts with Singleton Resources that must not support List https://google.aip.dev/156 so I'm not sure we can enforce "must" for List. I think it has to be the following:
```suggestion
- The resource API must support a [Get][] operation and should support a [List][] operation.
```

Reply:
I'll add the singleton exception, as decided in the other PR. let me know if you prefer something different.